vulcanai2/models/basenetwork.py

```diff
@@ -305,11 +361,12 @@ def train_epoch(self):
 
             # Forward + Backward + Optimize
             predictions = self(data)
-            loss = self.criterion(predictions, targets)
-            train_loss_accumulator += loss.item()
+
+            cur_loss = self.criterion(predictions, targets)
```

Comment:
can we have that as a `train_loss` instead?

Reply:
sure